### DISCUSSION TITLE
Lonhorn 1.5 sSupportes kubernetes-csi /external-snapshotter

### DISCUSSION THREAD

**Discussion by Delta1977** [longhorn/longhorn#6250] [Q&A]
Hi, 
one question.
Longhorn 1.4.x supports only kubernetes-csi/external-snapshotter Version 5.0.x.

Which kubernetes-csi/external-snapshotter version is Supported in Longhorn 1.5 ?

Thank you for answers

**Comment by innobead** [comment]
cc @ejweber

**Comment by ejweber** [comment]
Hello @Delta1977.

We have bumped our CSI sidecar to v6.2.1 and would prefer that version of the snapshot controller on the cluster as well.

https://longhorn.io/docs/1.5.0/snapshots-and-backups/csi-snapshot-support/enable-csi-snapshot-support/

That being said, the primary concern is that Longhorn v1.5.0 is no longer compatible with v1beta1 snapshot APIs. If you are using v1 snapshot APIs and your snapshot controller supports v1 snapshot APIs, there shouldn't be an issue.

https://longhorn.io/docs/1.5.0/snapshots-and-backups/csi-snapshot-support/enable-csi-snapshot-support/
